geoips/plugins/yaml/workflows/geostationary_dmw/windbarbs_dmw_high.yaml

```diff
@@ -0,0 +1,45 @@
+interface: workflows
+family: order_based
+name: ob_windbarbs_dmw_high
```

Comment:
This workflow is a port of existing product YAMLs ?  Currently unused (since all interfaces have not been implemented in the OBP), but this is what the individual "products" will look like within OBP (and referenced from the "abi.yaml" workflow above in the end).

Reply:
Yep. I wouldn't worry about this too much yet. The only workflow that i'm actually using for testing so far is `reader_test.yaml`.